apps/src/music/player/soundSub.js

```diff
@@ -40,7 +44,7 @@ function createAudioContext(desiredSampleRate) {
   return context;
 }
 
-function WebAudio() {
+function WebAudio(bpm) {
```

Comment:
Until now, this file has been agnostic to higher-level concepts like BPM, and has operated in the time domain.  Not a strong opinion, but I wonder whether we should maybe convert from BPM to the time domain higher in the stack?

Reply:
Yeah, I had a similar thought. We would still need some number to represent the amount of time in a beat in order to calculate delay and decay time though. Maybe we pass those in directly here and calculate them in a higher layer?

Reply:
Thanks for making changes here; it feels like we now have the right time domains in the right layers.  As a thought experiment, we can reuse our core audio components in a world without BPMs.